var/spack/repos/builtin/packages/geopm/package.py

```diff
@@ -18,22 +18,13 @@ class Geopm(AutotoolsPackage):
     msr-safe kernel module by your administrator."""
 
     homepage = "https://geopm.github.io"
-    url      = "https://github.com/geopm/geopm/releases/download/v0.4.0/geopm-0.4.0.tar.gz"
+    url      = "https://github.com/geopm/geopm/releases/download/v1.0.0/geopm-1.0.0.tar.gz"
     git      = "https://github.com/geopm/geopm.git"
 
     # Add additional proper versions and checksums here. "spack checksum geopm"
     version('develop', branch='dev')
     version('master', branch='master')
-    version('1.0.0-rc2', sha256='c6637df54728ded31fd682f39a07dffee45883f350e6dbd13e1496dd50243ffd',
-            url='https://github.com/geopm/geopm/releases/download/v1.0.0%2Brc2/geopm-1.0.0+rc2.tar.gz')
-    version('1.0.0-rc1', sha256='f8a2e5c384a15e9663f409de478b6372cd63e63a28d4701a33ac043fc27905e0', 
-            url='https://github.com/geopm/geopm/releases/download/v1.0.0-rc1/geopm-1.0.0+rc1.tar.gz')
-    version('0.6.1', sha256='0ca42853f90885bf213df190c3462b8675c143cc843aee0d8b8a0e30802b55a9')
```

Comment:
Usually we try to avoid removing old versions while updating recipes, or at least not all of them. Can you explain why the removal was necessary? Is the package following SemVer and 1.0 is a commitment on stability?

Reply:
Yes, Semver is used and 1.0 is a commitment to stability (both of the code and the interfaces).
Users should be using at least version 1.0. Everything prior can still be built from git directly but is not intended to be used on production systems. This is also why its scrapped from this, I can restore if wanted though